src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,52 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+import { Movie } from '../../types/Movie';
+
+export const MovieCard: React.FC<Movie> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+}) => (
+  <>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8" data-cy="MovieTitle">
+            {title}
+          </p>
+        </div>
+      </div>
+
+      <div className="content">
+        <p data-cy="MovieDescription">
+          {description}
+        </p>
+
+        <a
+          href={imdbUrl}
+          data-cy="MovieLink"
+        >
```

Comment:
Wrap if you have > 2 params || length > 80
Or, you can add target="_blank"
```suggestion
        <a href={imdbUrl} data-cy="MovieLink">
```